### PR TITLE
Adds several backdoors for monkeypatching

### PR DESCRIPTION
**Pull request by mikesol**
🙈 🙉 🙊 🐒

### CODE REVIEW

File: test/Main.purs [hunk 1]
```diff
@@ -32,6 +32,9 @@ import Test.Spec.Console (write)
 import Test.Spec.Reporter (consoleReporter)
 import Test.Spec.Runner (runSpec)
 import Type.Proxy (Proxy(..))
+import Unsafe.Coerce (unsafeCoerce)
+
+foreign import unsafeBackdoor :: MakeEvent -> Backdoor -> Effect MakeEvent
```
Comment (MonoidMusician) [inline test/Main.purs:37]: Looks like you forgot the JS file for this!

**Comment by MonoidMusician** [comment]
Otherwise looks fine to me. It's a bit messy but if it works!

File: test/Main.purs (hunk 1 above)
Comment (mikesol) [reply test/Main.purs:37]: Gloups, added.

**Comment by mikesol** [comment]
> Otherwise looks fine to me. It's a bit messy but if it works!

It works!

**Pull request merged**